data/src/main/java/nl/tehdreamteam/se42/data/conversation/hibernate/ConversationHibernateRepository.java

```diff
@@ -22,34 +21,22 @@ protected ConversationDAO createDao(EntityManager manager) {
 
     @Override
     public void save(Conversation conversation) {
-        Function<ConversationDAO, Void> function = dao -> {
-            dao.create(conversation);
-            return null;
-        };
-
-        super.performTransaction(function);
+        super.create(conversation);
     }
 
     @Override
-    public Conversation find(long id) {
-        Function<ConversationDAO, Conversation> function = dao -> dao.find(id);
-
-        return super.performTransaction(function);
+    public Conversation get(long id) {
+        return super.find(id);
     }
 
     @Override
     public void remove(long id) {
-        Conversation conversation = find(id);
+        Conversation conversation = get(id);
         remove(conversation);
     }
 
     @Override
     public void remove(Conversation conversation) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1185)